### PR TITLE
zephyr: Wrapped the RAM loading mode in Zephyr configs.

### PR DESCRIPTION
**Pull request by johan-ohman**
The RAM loading mode wasn't wrapped in Zephyr configs like the other boot modes.
Added a config to enable RAM loading and two configs to set IMAGE_EXECUTABLE_RAM_START
and IMAGE_EXECUTABLE_RAM_SIZE, respectively.

Signed-off-by: Johan Öhman <johan.ohman@softube.com>

### CODE REVIEW

File: boot/zephyr/Kconfig [hunk 1]
```diff
@@ -234,8 +234,25 @@ config BOOT_DIRECT_XIP
...
+	  If y, mcuboot selects the newest valid image based on the image version
+	  numbers, thereafter the selected image is copied to RAM and executed from
+	  there. For this reason, the image has to be linked to be executed from RAM.
+	  The address that the image is copied to is specified using the load-addr
+	  argument to the imgtool.py script which writes it to the image header.
+
 endchoice
 
+if BOOT_RAM_LOAD
+config BOOT_IMAGE_EXECUTABLE_RAM_START
```
Comment (nvlsianpu) [inline boot/zephyr/Kconfig:249]: values for `IMAGE_EXECUTABLE_RAM_START` and `IMAGE_EXECUTABLE_RAM_SIZE` macros should have value source in DTS, similar as fixed-partitions.

**Review by nvlsianpu — COMMENTED** [review]
Hi.

Zephyr-rtos is using this repo as stable module fork. Contribution to the origin fork is preferred.
(I'm doing synchronization quite frequently)

**Comment by johan-ohman** [comment]
> Hi.
> 
> Zephyr-rtos is using this repo as stable module fork. Contribution to the origin fork is preferred. (I'm doing synchronization quite frequently)

Ok! I guess I'll just close this pull request then. I'll take your comment about the values belonging in the DTS into consideration before I make the new pull request in the right repo.
Is it to https://github.com/mcu-tools/mcuboot/tree/main that I should do the new one?

**Comment by nvlsianpu** [comment]
Yes, https://github.com/mcu-tools/mcuboot/tree/main is the right place.

**Pull request closed without merging**